docs/framework/migration-guide/retargeting/workflow_30_types_are_obsolete.md

```diff
@@ -0,0 +1,10 @@
+### WorkFlow 3.0 types are obsolete
+
+|   |   |
+|---|---|
+|Details|Windows Workflow Foundation (WWF) 3.0 APIs (those from the System.Workflow namespace) are now obsolete.|
+|Suggestion|New WWF 4.0 APIs (in System.Activities) should be used instead. An example of using the new APIs can be found <a href="https://docs.microsoft.com/en-us/dotnet/articles/framework/windows-workflow-foundation/how-to-update-the-definition-of-a-running-workflow-instance">here</a> and further guidance is available <a href="http://blogs.msdn.com/b/workflowteam/archive/2012/02/08/deprecatingwf3.aspx">here</a>. Alternatively, since the WWF 3.0 APIs are still supported, they may be used and the build-time warning avoided either by suppressing it or by using an older compiler.|
```

Comment:
here shouldn't be used as the link text and links to articles inside the repo should be relative paths